buddyboss/buddyboss-platform
# PROD-668 #3220: Woocommerce product search widget shows hidden product when BB platform is activated - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3220  .

### How to test the changes in this Pull Request:

1. Add woocommerce product search widget in footer or sidebar
2. Make sure you have a product with catalog visibility hidden.
3. Now search for that product in product search widget
4. See the product is not showing in search result.

### Proof Screenshots or Video

https://share.vidyard.com/watch/CHh8sMJd7EmimzBchAH96G?

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
